src/hdmf/validate/validator.py

```diff
@@ -152,6 +152,7 @@ class ValidatorMap(object):
     def __init__(self, **kwargs):
         ns = getargs('namespace', kwargs)
         self.__ns = ns
+        self._ns = ns
```

Comment:
What was this addition for?

Reply:
remnants of debugging... I'll remove that